### PR TITLE
Adding augment_lr doc string

### LINKED ISSUE #4: Function 3 Docs - augment_lr()

### PR DESCRIPTION
**Pull request by heidi-ye**

### CODE REVIEW

File: R/sptidy.R [hunk 1]
```diff
@@ -32,12 +32,29 @@ tidy_lr <- function() {
...
 
-#' Title
+#' Augmented Output for Tidymodel's Linear Regression
 #'
-#' @return
+#' Adds two columns to the original data of the scikit learn's linear regression model. This includes predictions and residuals.
+#'
+#' @params model _lm
+#' @params X data.frame
+#' @params y data.frame
```
Comment (xudong-Y) [inline R/sptidy.R]:
```suggestion
#' 
```
Hi Heidi! I think for tidymodels we do not need a y dataframe to fit the model.

File: R/sptidy.R [hunk 2]
```diff
@@ -32,12 +32,29 @@ tidy_lr <- function() {
...
+#' library(tidyverse)
+#' library(tidymodels)
+#' # Load data
+#' data("longley")
+#' # Fit linear regression
+#' my_lr <- linear_reg() %>%
+#' set_engine("lm") %>%
+#' fit(Employed~., data = longley)
+#' # Return augmented dataframe
+#' augment_lr(my_lr,X,y)
```
Comment (xudong-Y) [inline R/sptidy.R]:
```suggestion
#' augment_lr(my_lr,longley)
```

File: R/sptidy.R (hunk 1 above)
Comment (heidi-ye) [reply R/sptidy.R]: Good to know! Thanks!

**Comment by xudong-Y** [comment]
Thanks Heidi! Merging in.

**Pull request merged**